x/concentrated-liquidity/types/expected_keepers.go

```diff
@@ -19,6 +19,7 @@ type AccountKeeper interface {
 // BankKeeper defines the banking contract that must be fulfilled when
 // creating a x/concentrated-liquidity keeper.
 type BankKeeper interface {
+	GetBalance(ctx sdk.Context, addr sdk.AccAddress, denom string) sdk.Coin
 	GetAllBalances(ctx sdk.Context, addr sdk.AccAddress) sdk.Coins
```

Comment:
Is `GetAllBalances` still needed or can be removed?

Reply:
Good question, let me check

Reply:
We still need it for a query